apps/modernization-ui/src/apps/page-builder/pages/EditPage/PageInfo/PageInfo.tsx

```diff
@@ -0,0 +1,170 @@
+import './PageInfo.scss';
+import { Button, Icon, Pagination } from '@trussworks/react-uswds';
+import React, { useContext, useEffect, useState } from 'react';
+import { PageControllerService, PageInformationService, PagesResponse } from '../../../generated';
+// import { downloadAsCsv } from '../../../../../utils/downloadAsCsv';
+import { UserContext } from '../../../../../providers/UserContext';
+import { useNavigate } from 'react-router-dom';
+// import { saveAs } from 'file-saver';
+
+type Props = {
+    question?: any;
+    page: PagesResponse;
+};
+
+const pageSize = 3;
+export const PageInfo = ({ page }: Props) => {
+    const { state } = useContext(UserContext);
+    const [activeTab, setActiveTab] = useState('Details');
+    const [totalResults, setTotalResults] = useState(4);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [pageHistory, setPageHistory] = useState([
+        { publishVersionNbr: '', lastUpdatedDate: '', lastUpdatedBy: '', notes: '' }
+    ]);
+    const navigate = useNavigate();
+    const token = `Bearer ${state.getToken()}`;
+
+    const handleDownloadCSV = async () => {
+        try {
+            PageControllerService.downloadPageMetadataUsingGet({
+                authorization: token,
+                waTemplateUid: page.id
+            }).then((data) => {
+                const fileType = 'application/xlsx'; // 'text/vnd.openxmlformats-officedocument.spreadsheetml.sheet';
+                // @ts-ignore
+                const blob = new Blob([data], { type: fileType });
+                const url = window.URL.createObjectURL(blob);
+                const a = document.createElement('a');
+                a.href = url;
+                a.download = 'PageMetadata.xlsx';
+                a.click();
+                // downloadAsCsv({
+                //     data,
+                //     fileName: '1PageMetadata.xlsx',
+                //     fileType
+                // });
+            });
+            // @ts-ignore
+            // const blob = new Blob([data], { type: fileType });
+            // saveAs(blob, 'PageMetadata.xlsx');
+        } catch (error) {
+            console.log(error);
+        }
+    };
+
+    const fetchPageHistory = async () => {
+        try {
+            const rep: any = await PageInformationService.getPageHistory({
+                authorization: token,
+                page: page.id
+            });
+            setPageHistory(rep);
+            setTotalResults(rep.length);
+        } catch (error) {
+            console.log(error);
```

Comment:
Is there a reason why we are not notifying the user of the error? Can we just remove the catch here?